.github/prompts/benchmark.prompt.md

```
@@ -0,0 +1,165 @@
+- **Namespaces must NOT end with `.Benchmarks`.**  
+  They must mirror the production assembly’s namespace.
+
+  Example:  
+  If benchmarking a type inside `Cuemon.Security.Cryptography`, then:
+
+  ```csharp
+  namespace Cuemon.Security.Cryptography
+  {
+      public class Sha512256Benchmark { … }
+  }
+````
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)